doc/nrf/libraries/others/sfloat.rst

```diff
@@ -0,0 +1,61 @@
+.. _lib_sfloat:
+
+Short float (SFLOAT)
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
```

Comment:
As in the template, we need a few words about the library before the Overview section:

"Use this first section to give a short description (1 or 2 sentences) of the library, indicating what it does and why it should be used."

Reply:
Rearranged the documentation and wrote some additional content to meet these criteria